### PR TITLE
Added menu arrow after nextcloud/server#3282

### PR DESCRIPTION
**Pull request by skjnldsv**
Following nextcloud/server#3282

capture d ecran_2017-01-26_15-59-02

@nextcloud/designers

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -173,8 +173,18 @@ img.notification-icon {
 
 .notification-container {
 	box-shadow: 0 1px 10px rgba(50, 50, 50, .7);
-	overflow: auto;
-	overflow-x: hidden;
+}
+
+/* Menu arrow */
+.notification-container:after {
+	right: 53px;
```
Comment (MorrisJobke) [inline css/styles.css]: This needs to be done in JS I think:

bildschirmfoto 2017-01-26 um 12 46 48
bildschirmfoto 2017-01-26 um 12 47 01
bildschirmfoto 2017-01-26 um 12 47 25

Comment (skjnldsv) [inline css/styles.css]: How did I missed that... On it... -_-

**Comment by skjnldsv** [comment]
@MorrisJobke Fixed! 🎉

**Comment by skjnldsv** [comment]
Narrow screen is broken

**Comment by nickvergessen** [comment]
> bildschirmfoto vom 2017-01-27 10-00-34

**Comment by skjnldsv** [comment]
@nickvergessen That what I said! ^^

**Comment by nickvergessen** [comment]
I just reposted it, because morris canceled it.

**Comment by skjnldsv** [comment]
I don't think we can do without js this time :/
Except if we use something else instead of the username on narrow screen (which could be great)
refs https://github.com/nextcloud/server/issues/3273#issuecomment-275748570

**Comment by jancborchardt** [comment]
@skjnldsv check out https://github.com/nextcloud/server/pull/4074 - with that the name is removed, so we can set a fixed spot for the triangle. :)

**Comment by jancborchardt** [comment]
Fixed:
capture du 2017-03-27 13-23-06

Please review @skjnldsv @nickvergessen :)

**Pull request merged**